src/app/Main/Header/Export/Header-ExportContainer.component.tsx

```diff
@@ -38,53 +42,74 @@ export function HeaderExportContainerComponent(): JSX.Element {
     const [setting, setSetting] = useState<ExportInterface>(defaultExport);

```

Comment:
only state really needed here is stat and raw flags, all the other things are only used by fetching function in a stateless manner